src/Angor/Client/NetworkConfiguration.cs

```diff
@@ -14,6 +15,8 @@ public class NetworkConfiguration : INetworkConfiguration
     public static short NostrEventIdKeyType = 1; //TODO David use an enum for this?
 
     public int GetAngorInvestFeePercentage => AngorInvestFeePercentage;
+    public static IClientStorage? Storage;
```

Comment:
we do not reference storage in this class, you can only set params here form outside the class